docs/Networking.asciidoc

```diff
@@ -52,6 +52,102 @@ or Wicked way:
 </interface>
 ---------------
 
+## TAP with Open vSwitch
+
+The recomended way to configure the network for TAP devices is using Open vSwitch.
+There is a support service +os-autoinst-openvswitch.service+ which sets vlan number
+of Open vSwitch ports based on +NICVLAN+ variable - this separates the groups of
```

Comment:
Some note about NICVLAN, please. Is set automatically or should admin handle that?